### PR TITLE
Init project phase 1

### PR DESCRIPTION
**Pull request by mpereannor**
# Description

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe steps taken to ensure this feature is functional and does not break other features:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation, if relevant
- [ ] I have run the app using my feature and ensured that no functionality is broken
- [ ] My changes generate no new warnings
- [ ] There are no merge conflicts

### CODE REVIEW

**Review by chineducn — APPROVED** [review]

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <!--
+      manifest.json provides metadata used when your web app is installed on a
+      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
+    -->
```
Comment (RodPaDev) [inline public/index.html:16]: This should be removed for production

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+@keyframes App-logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
```
Comment (RodPaDev) [inline src/App.css:38]: all .css files should be deleted since they are no use to us

File: src/index.css [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+code {
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+    monospace;
+}
```
Comment (RodPaDev) [inline src/index.css:13]: once again this should be deleted, if we need to import fonts or any global styles, we should use` createGlobalStyle` dependency from` styled-components`

**Review by RodPaDev — APPROVED** [review]
Seems good to me, just some minor linting for future proof

**Pull request merged**